src/cfnlint/custom_rules.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import cfnlint
+import cfnlint.customRules.Operators
+
+LOGGER = logging.getLogger(__name__)
+Operator = {'EQUALS': lambda x, y: cfnlint.customRules.Operators.equalsOp(x, y)}
+
+def check(filename, template):
+    """ Process custom rule file """
+    matches = []
+    with open(filename) as customRules:
+        count = 1
+        for line in customRules:
+            LOGGER.debug('Processing Custom Rule Line %d', count)
+            line = line.replace('"', '')
+            rule = line.split(' ')
+            if len(rule) == 4 and rule[0][0] != '#':
+                resource_type = rule[0]
+                operator = rule[2]
+                try:
+                    result = Operator[rule[2]](rule, template.get_resource_properties([resource_type]))
+                    if result != operator:
+                        matches.append(cfnlint.rules.Match(count, '0', '0', '0', filename, CustomRule('E9999'), result, None))
+                except KeyError as e:
+                    matches.append(cfnlint.rules.Match(count, '0', '0', '0', filename, CustomRule('E9999'), 'Error - Invalid Operator', None))
```

Comment:
Nice catch! Might help speed up debugging to add more context to the error message:
```
= not in supported operators: [==, !=] at AWS::SNS::Topic TopicName = "A" (custom_rules.txt line 77)
```

Reply:
Sounds like a smart idea, I'll get it pushed with the next commit!